app/controllers/pbs/events_controller.rb

```diff
@@ -72,6 +73,22 @@ def assign_contact_attrs_with_supercamp_flag
     contact_attrs
   end
 
+  def preview_camp_application_validation
+    return unless entry.is_a?(Event::Campy)
+    return unless [entry.leader, entry.coach].include? current_person
+    return if entry.camp_submitted?
+
+    original_camp_submitted_at = entry.camp_submitted_at
+    entry.camp_submitted_at = Time.zone.now.to_date
+    if entry.valid?
+      flash.now[:notice] = "#{I18n.t('events.create_camp_application.flash.preview_success')}"
+    else
+      flash.now[:warn] = "#{I18n.t('events.create_camp_application.flash.preview')}" \
+                        "<br />#{entry.errors.full_messages.join('; ')}"
```

Comment:
Wäre vielleicht ein richtiger Satz besser?

```suggestion
                        "<br />#{entry.errors.full_messages.to_sentence}"
```

Reply:
Finde ich an und für sich gut, aber das spiegelt halt die Nachricht wieder die bei einem fehlgeschlagenen Button-Klick durch den Coach angezeigt wird. Also werde ich gleich beide zu Sätzen ändern.